fendermint/testing/graph-test/README.md

```diff
@@ -0,0 +1,20 @@
+# Integration test with The Graph
+
+## Prerequisite 
+0. Install `npm` if you haven't done so already.
+1. Instsall `graph` CLI.
+```bash
+npm install -g @graphprotocol/graph-cli
```

Comment:
Doesn't [this call](https://github.com/consensus-shipyard/ipc/pull/592/files#diff-732bb321308006d735c68b139c7058692df5f638eb4abdc5f99e4f2041f7c151R55) to `npm install` take care of this?

Reply:
You are right! I changed to use `npx` to execute local binaries instead.